lib/components/molecules/Accordian/Accordian.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+/**
+ *
+ * Accordian
+ *
+ */
+import { PureComponent } from 'react';
+import styled from 'styled-components';
+
+import { AccordianWrapper, AccordianHeader, AccordianBody } from './Accordian.style';
+import type { Props, State } from './types';
+
+class Accordian extends PureComponent<Props, State> {
+  static defaultProps = {};
+
+  state: State = {
+    isOpen: false,
+  };
+
+  toggleAccordianHandler = () => {
+    const doesShow = this.state.isOpen;
+    this.setState({ isOpen: !doesShow });
+  };
+
+  render() {
+    const { accordianHeader, accordianBody, className } = this.props;
+
+    return (
+      <AccordianWrapper>
+        <AccordianHeader
```

Comment:
AccordianHeader will not be DIV but a summary tag, you should built it as separate component.